### PR TITLE
Updated wording around creating a secret

### PR DESCRIPTION
**Pull request by mpetason**
Updating some of the wording around creating a secret in the Tekton guide. This may make it read a little bit better.

### CODE REVIEW

File: content/guides/ci-cd/tekton-gs-p1.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ tekton-pipelines-webhook-69796f78cf-b28z4      1/1     Running             0
 tekton-pipelines-controller-6d55778887-df59t   1/1     Running             0          13s
 ```
 
-Finally, in a couple of the examples, you'll be pushing up container images to Docker Hub, so create a secret that Tekton can use to log in to Docker Hub with, substituting the placeholder values with your own:
+Finally, in a couple of the examples, you'll be pushing container images to Docker Hub, so create a secret that can be used by Tekton to log in to Docker Hub. Substitute the placeholder values with your own:
```
Comment (anthonyvetter) [inline content/guides/ci-cd/tekton-gs-p1.md:65]:
```suggestion
Finally, in a couple of the examples, you'll be pushing container images to Docker Hub. Create a secret that Tekton can use to log in to Docker Hub. Substitute the placeholder values with your own:
```

**Comment by anthonyvetter** [comment]
Thank you for the contribution! Sorry for the delay. I had our Copy Editor look at your suggestion and I have added their suggestion. Let me know what you think.

**Comment by anthonyvetter** [comment]
I have not heard back from you so I have committed these changes and am merging them now. Thank you again for your contribution!

**Comment by mpetason** [comment]
Sorry for the delay! Thank you for the updates! Everything looks great.

**Pull request merged**